lxd/main_init.go

```diff
@@ -45,7 +46,7 @@ func (c *cmdInit) Command() *cobra.Command {
 	cmd.Example = `  init --preseed
   init --auto [--network-address=IP] [--network-port=8443] [--storage-backend=dir]
               [--storage-create-device=DEVICE] [--storage-create-loop=SIZE]
-              [--storage-pool=POOL] [--trust-password=PASSWORD]
+              [--storage-pool=POOL] [--trust-password=PASSWORD] [--trust-ca-certificates]
```

Comment:
I wouldn't make this a `lxd init` option.

Reply:
It's so uncommon a setup that it can definitely be enabled by hand through `lxc config`.

Reply:
Fixed